hack/run

```diff
@@ -1,5 +1,16 @@
 #!/usr/bin/env bash
 
+RUN()
+{
+  echo "running $@"
+  if [ -n "$HSM" ]; then
+     go build -o signing-server "$1"
+     sudo ./signing-server "${@:2}"
```

Comment:
Why is this running with `sudo`?

Reply:
If I had to guess: might the port default to 80/443? 🙈

Reply:
hmmm, maybe but I'm not convinced. :D

Reply:
no this is running in sudo because softhsm cannot be targeted as a module without running the app as admin. Thats why this exists. other HSMs (especially ones backed by an API) do not need to have administrative privileges